### ISSUE TITLE
Not working with with Xcode 7

### ISSUE THREAD

**Issue by hardikdevios** [RolandasRazma/RRConstraintsPlugin#6]
please update the plugin

**Comment by RolandasRazma** [comment]
hi, Xcode vastly improved work with constraints. Is there anything in specific you are missing from plugin?

**Comment by hardikdevios** [comment]
@RolandasRazma yes mate in Terms of Xcode they have much improved now but the thing is your functionality which you have mentioned seems to be not working in Xcode 7 for eg in Align panel you have mentioned distributed Horizontally and vertically , but when you check in Xcode 7 you won't see any option to distribute the Views same as embed view is not saving the constraints when i click on embeded in view all the constraints  is getting removed so can you please check in Xcode 7 i hope there must be small tweak let me know when you update for Xcode 7

**Comment by RolandasRazma** [comment]
@hardikdevios plugin doesn't work with Xcode 7 at all - I know that. Just wondering witch parts of it you using and wish they would work.

**Comment by hardikdevios** [comment]
My main requirement is the 
- Embedding in View should maintain the constraint  
  is it possible with the Xcode 7 ?

**Comment by RolandasRazma** [comment]
at the moment plugin is not compatible with Xcode 7 and none of the functions working

**Comment by hardikdevios** [comment]
@RolandasRazma  ya but whats your future plan are you going to make it compatible or you are going to stop the release ?

**Comment by RolandasRazma** [comment]
Hi, at the moment I have no plans updating. 
I started it on Xcode 5 when constraint support was lacking, but nowadays Xcode does most of the things itself. It is true that some features are missing and it would be nice to update plugin, but I don't have enough time at the moment to do that.

**Comment by hardikdevios** [comment]
ohk No issue mate cheers :+1:
